### PR TITLE
Add Numpy Flexible example

### PR DESCRIPTION
**Pull request by waprin**

### CODE REVIEW

File: appengine/flexible/numpy/main.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
```
Comment (jonparrott) [inline appengine/flexible/numpy/main.py]: 2016

File: appengine/flexible/numpy/main.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    y = np.array([[5, 6], [7, 8]])
+
+    return_str += 'x: {} , y: {}<br />'.format(str(x), str(y))
+
+    # Multiply matrices
+    return_str += 'x dot y : {}'.format(str(np.dot(x, y)))
+    return return_str
+
+
+@app.route('/')
```
Comment (jonparrott) [inline appengine/flexible/numpy/main.py]: You can probably remove this and just make `calculate` the root url.

Comment (waprin) [inline appengine/flexible/numpy/main.py]: sgtm but I feel like this wasn't done in DPE bot so wasn't sure what our standard was (I know that's different repo)

File: appengine/flexible/numpy/main.py (hunk 1 above)
Comment (waprin) [inline appengine/flexible/numpy/main.py]: done

File: appengine/flexible/numpy/main.py (hunk 2 above)
Comment (jonparrott) [inline appengine/flexible/numpy/main.py]: DPEBot's root URL will eventually be a UI to add him to a repo. :)

**Comment by jonparrott** [comment]
@waprin can you rebase on master?

**Comment by jonparrott** [comment]
@dpebot LGTM

**Comment by jonparrott** [comment]
@dpebot merge when travis passes

**Comment by jonparrott** [comment]
@waprin looks like you need to run `nox -s reqrollup` to fixup the test dependencies.

**Pull request merged**